extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -328,7 +346,7 @@ public function loadDataFromRevDataProvider() {
 				'parent_page_id' => '2147',
 				'msg_author_id' => 0,
 				'msg_author_username' => '65.19.148.1',
-				'msg_author_displayname' => 'A Wikia contributor',
+				'msg_author_displayname' => static::A_FANDOM_USER,
 				'wall_username' => 'Garthwebb',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wall_username" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)